lib/auth/oidc.go

```diff
@@ -57,6 +57,50 @@ func (a *Server) UpsertOIDCConnector(ctx context.Context, connector types.OIDCCo
 	return nil
 }
 
+// UpdateOIDCConnector updates an existing OIDC connector.
+func (a *Server) UpdateOIDCConnector(ctx context.Context, connector types.OIDCConnector) (types.OIDCConnector, error) {
+	updated, err := a.Services.UpdateOIDCConnector(ctx, connector)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	if err := a.emitter.EmitAuditEvent(ctx, &apievents.OIDCConnectorCreate{
+		Metadata: apievents.Metadata{
+			Type: events.OIDCConnectorCreatedEvent,
+			Code: events.OIDCConnectorCreatedCode,
+		},
+		UserMetadata: authz.ClientUserMetadata(ctx),
+		ResourceMetadata: apievents.ResourceMetadata{
+			Name: connector.GetName(),
+		},
+	}); err != nil {
+		log.WithError(err).Warn("Failed to emit OIDC connector create event.")
```

Comment:
```suggestion
		log.WithError(err).Warn("Failed to emit OIDC connector update event.")
```

Reply:
Nevermind, I see now that there isn't an actual update event and they both use the create event. Ignore